docs/source/format/Columnar.rst

```diff
@@ -790,7 +790,7 @@ type.
 **Example Layout: ``Struct<VarBinary, Int32>``**
 
 The layout for ``[{'joe', 1}, {null, 2}, null, {'mark', 4}]``, having
```

Comment:
Would you mind change the string to 

```
``[{'joe', 1}, {'alice', 2}, null, {'mark', 4}]`
```

? Seems changing this also makes the result works?

Reply:
Let me investigate `carefully` first. I will set this as draft first. 
Once I have thought it through, I will ping you. 
Thank you very much!

Reply:
I think there seems to be no problem with this example, as seen from the following chapter `Struct Validity`
<img width="753" alt="image" src="https://github.com/user-attachments/assets/4f9a9a51-88eb-42e2-a27a-54a90c33d529" />

So, I will close it.
cc @mapleFU